nuxeo-drive-client/tests/common_unit_test.py

```diff
@@ -525,11 +518,6 @@ def tearDown(self):
     def tearDownApp(self, server_profile=None):
         if self.tearedDown:
             return
-        if sys.exc_info() != (None, None, None):
-            self.generate_report()
-        elif self.result is not None:
-            if hasattr(self.result, "wasSuccessful") and not self.result.wasSuccessful():
-                self.generate_report()
```

Comment:
Why remove the report generation? It's useful for debugging as it includes the database for instance.